src/dataServices/dynamoDbBundleService.test.ts

```diff
@@ -450,4 +450,55 @@ describe('atomicallyReadWriteResources', () => {
             await runUpdateTest(true);
         });
     });
+
+    describe('Update as Create Cases', () => {

```

Comment:
These look great could you run: `yarn run test --coverage` this will show any uncovered lines. You can just share the `src/dataServices/dynamoDbBundleService.ts` row if you want

Reply:
There are some uncovered lines, overall pretty good coverage:
`dynamoDbBundleService.ts           |   85.53 |    75.38 |   88.89 |   85.81 | 54,60,76-80,109-113,162-164,188,323-328,354-355,373-375,428-429`

Reply:
Great your changed lines are covered :)